frontend/src/container/LogsExplorerList/InfinityTableView/index.tsx

```diff
@@ -65,6 +65,8 @@ const InfinityTable = forwardRef<TableVirtuosoHandle, InfinityTableProps>(
 			...tableViewProps,
 			onClickExpand: onSetActiveLog,
 			onOpenLogsContext: handleSetActiveContextLog,
+			activeLog: activeLog as ILog,
```

Comment:
Rather than type force, can we use `defaultTo` from `lodash`

Reply:
updated